src/main/webapp/i18n/en/exercise.json

```diff
@@ -45,6 +45,8 @@
             "sampleSolution": "Example Solution",
             "gradingInstructions": "Grading Instructions",
             "structuredGradingInstructions": "Structured Grading Instructions",
+            "addGradingCriteria": "Add Grading Criteria",
+            "deleteGradingCriteria": "Delete This Criteria",
```

Comment:
I think it should be either singular "Delete This Criterion" or plural "Delete These Criteria"